conf/json/schema/api/schemas.json

```diff
@@ -1021,9 +1021,13 @@
         "operator": {
           "type": "string",
           "enum": [
+            "<",
+            "<=",
             "=",
             ">",
-            "<"
+            ">=",
+            "in",
+            "instanceOf"
```

Comment:
Added `instanceOf` as enumeration literal for `PrimitiveConstraint#operator` - not currently supported in pilot implementation